project/Http4sBuild.scala

```diff
@@ -22,7 +22,7 @@ object Http4sBuild {
    * scalaz-7.2 "a" versions for 0.15.x and 0.16.x.
    */
   def docExampleVersion(currentVersion: String) = {
-    val MilestoneVersionExtractor = """(0).(16|17).(0)-SNAPSHOT""".r
+    val MilestoneVersionExtractor = """(0).(16|17).(0)a?-SNAPSHOT""".r
```

Comment:
This kind of regex makes me excited about our future and regretful of our present! :)